Balanced Clean Install.ipynb

```diff
@@ -1912,7 +1916,7 @@
     "    .nid(NID)\\\n",
     "    .nonce(100)\\\n",
     "    .method(\"setMarketName\")\\\n",
-    "    .params({'_pid':3, '_name': 'BALNbnUSD'}) \\\n",
+    "    .params({'_id':3, '_name': 'BALNbnUSD'}) \\\n",
```

Comment:
@gedanziger can you updated the _name in the params here

Reply:
ok will do it

Reply:
updating pool names after a bit of discussion, so it will reflect same as the frontend.

Reply:
Updated in rebase, all market names in jupyter, dex, and governance have been updated according to the following:

`sICX/ICX`
`sICX/bnUSD`
`BALN/bnUSD`

After discussing with @dbrehmer this seemed to be the best solution.